### DISCUSSION TITLE
Expressões regulares

### DISCUSSION THREAD

**Discussion by rafamalta** [balta-io/forum#189] [Tira Dúvidas] [answered]
Boa noite professor. No curso 'Fundamentos do C#', módulo 'Mão na massa: Editor HTML', aula 'Visualizador' você fala que 'terá uma parte para falar sobre elas'. Poderia informar em qual aula é visto essa parte?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rafamalta, sobre qual assunto seria?

**Reply by rafamalta** [reply]
Perdão, esqueci de mencionar.  O assunto é sobre Expressões Regulares.
